src/pages/auth/auth.ts

```diff
@@ -29,6 +31,27 @@ export class AuthPage {
     ionViewDidEnter(): void {
         if (this.auth.isAuthenticated()) {
             this.navCtrl.setRoot(AuthDetailsPage);
+        } else {
+            const configKeys: string[] = Object.keys(this.auth.getConfig()).toString().split(',');
+            console.log(configKeys.length);
+            if (configKeys.length <= 1 || configKeys.indexOf('url') === -1) {
+                this.configValid = false;
+                this.showInvalidAuthConfigAlert();
+            } else {
+                this.configValid = true;
+            }
         }
     }
+
+    showInvalidAuthConfigAlert(): void {
```

Comment:
I was wondering, would it be better if we created a validate service for these functions, and then just call on them where we need them.

What do you think @pawelpaszki @StephenCoady

Reply:
@ciaranRoche you are probably right. I will make the required changes